src/main/java/org/threadly/examples/features/package-info.java

```diff
@@ -0,0 +1,8 @@
+/**
+ * 
+ */
+/**
+ * @author sreid8
+ *
+ */
+package org.threadly.examples.features;
```

Comment:
I don't like to have the author tags in the package javadocs.  Could you add a small one or two sentence description here, for example:

A collection of small examples using specific threadly features.